hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -356,6 +362,12 @@ public String getLockOwnerInfo(Data key) {
     @Override
     public Record loadRecordOrNull(Data key, boolean backup) {
         Record record = null;
+
+        if (logger.isFineEnabled()) {
+            Object keyAsObj = mapServiceContext.toObject(key);
+            logger.fine("Loading record for map " + name + " with key " + keyAsObj + " " + getStateMessage());
```

Comment:
isn't it to aggressive to log every load attempt? Is it really required?

Reply:
It is, especially on fine level. Removed it together with finest logging each batch in `loadFromAll()`.